### PR TITLE
UI improvements/approve buttons

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Combines approval button under one dropdown.
- Depends on merge of https://github.com/Azure/azure-sdk-tools/pull/4783
AprrovalStuff
ApproveDropdown
- Should resolve https://github.com/Azure/azure-sdk-tools/issues/4627

### CODE REVIEW

**Comment by maririos** [comment]
@chidozieononiwu do you have a screenshot on how this change looks now? or is it deployed into the apiviewuat instance?

**Comment by chidozieononiwu** [comment]
> @chidozieononiwu do you have a screenshot on how this change looks now? or is it deployed into the apiviewuat instance?

I have it deployed to https://apiviewuat.azurewebsites.net/

**Comment by praveenkuttappan** [comment]
Please make a change as per suggestion from Ted to change the first approve button label to `Approval for GA`. Otherwise looks fine.

**Review by praveenkuttappan — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml [hunk 1]
```diff
@@ -33,22 +33,128 @@
...
+                            }
+                            itemRendered = true;
+                        }
+                    </form>
+                    @if (itemRendered)
+                    {
+                        <small class="form-text text-muted mx-4 my-1">Approval for Current Revision of the API.</small>
+                        <div class="dropdown-divider"></div>
+                        itemRendered = false;
                    }
```
Comment (konrad-jamrozik) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml:121]:
```suggestion
                        itemRendered = false;
                    }
```

Comment (chidozieononiwu) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml:121]: Its strange but I don't see this on my local machine. I wonder what's going on here.

**Pull request merged**